src/oas/operation.ts

```diff
@@ -89,6 +89,7 @@ export const transformOasEndpointOperation: TranslateFunction<
         parentId: serviceId,
         method,
         path: name,
+        key: key ?? 'unknown',
```

Comment:
Or should we generate random string in that case?

Reply:
i think it would have to be something stable. i'm kind of thinking this should throw but @P0lip might have a suggestion

Reply:
You can leave an empty key. That's what we do for schemas

Reply:
Updated